services/main-frontend/src/components/page-specific/organizations/index/CreateOrganizationPopup.tsx

```diff
@@ -0,0 +1,144 @@
+import { css } from "@emotion/css"
+import React from "react"
+import { useTranslation } from "react-i18next"
+
+import StandardDialog from "@/shared-module/common/components/dialogs/StandardDialog"
+
+interface CreateOrganizationPopupProps {
+  show: boolean
+  setShow: React.Dispatch<React.SetStateAction<boolean>>
+  name: string
+  setName: React.Dispatch<React.SetStateAction<string>>
+  visibility: string
+  setVisibility: React.Dispatch<React.SetStateAction<string>>
+  handleCreate: () => void
+  slug: string
+  setSlug: React.Dispatch<React.SetStateAction<string>>
+}
+
+const CreateOrganizationPopup: React.FC<CreateOrganizationPopupProps> = ({
+  show,
+  setShow,
+  name,
+  setName,
+  visibility,
+  setVisibility,
+  handleCreate,
+  slug,
+  setSlug,
+}) => {
+  const { t } = useTranslation("main-frontend") as { t: (key: string) => string }
```

Comment:
Don't do the `{ t: (key: string) => string }` as it just disables type checking for the translations. If typescript does not pick your new translation, you need to

a) Check that you have `bin/shared-module-sync-watch` running in a terminal
b) Press ctrl-shift-p on VSCode, search for restart ts server, and select that one

Reply:
Also no need to specify "main-frontend". Just `const { t } = useTranslation("main-frontend")` suffices.

Protip: we have a snippet for this in VSCode. If you type `ut`, it will autocomplete to this line. Also if, you're trying to add a translated string in jsx context, try typing `jt`.